source/keyboard-shortcuts.txt

```diff
@@ -0,0 +1,247 @@
+==================
+Keyboard Shortcuts
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Keyboard shortcuts enable you to easily navigate |compass-short|.
+
+.. list-table::
+   :widths: 20 30 25 25
+   :header-rows: 1
+
+   * - Category
+     - Description
+     - Windows
+     - Mac
+
+   * - Menu
+     - Online help
+     - ``F1``
+     - ``F1``
+
+   * - Menu
+     - Quit application
+     - ``Ctrl`` + ``Q``
+     - ``Cmd`` + ``Q``
+
+   * - Menu
+     - Show settings
+     - ``Ctrl`` + ``,``
+     - ``Cmd`` + ``,``
+
+   * - Menu
+     - Hide window
+     - ``Ctrl`` + ``H``
+     - ``Cmd`` + ``H``
+
+   * - Menu
+     - Hide other windows
+     - ``Ctrl`` + ``Shift`` + ``H``
+     - ``Cmd`` + ``Shift`` + ``H``
+
+   * - Menu
+     - Opens a new Compass window
```

Comment:
To keep the same convention as the other Descriptions:
```suggestion
     - Open a new Compass window
```

Reply:
Updated.